opentracing/ext/tags_test.go

```diff
@@ -0,0 +1,35 @@
+package ext_test
+
+import (
+	"testing"
+
+	"github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
maybe we should rename `examples` to `testing`... seems weird to have anything under `api-golang/opentracing/` depending on anything under `api-golang/examples/`, even if it's just a test.